article/sections/_7_latent_confounding.tex

```diff
@@ -1,5 +1,224 @@
+Without accounting for this latent confounder, we risk getting very biased estimates 
+of the effects of our treatment variable of interest, created by the 
+induced variations by the confounder in both the treatment and the outcome 
+variable. 
+
+
+
+The problem of latent confounding and ommitted variable bias is widely 
+acknowledged in the transportation literature, and demand modelers can draw on 
+a list of method to account for confouding in some specific circumstances. 
+Integrated choice and latent variable (ICLV) models are a way to account for 
+the effect of unobserved attitudinal variables which may affect the selection 
+of some individuals into some treatment level, as well as an outcome of 
+interest. For example, a person's unobserved beliefs and attitudes towards 
+being environmentelly friendly may both affect whether she chooses to bike to 
+work, as well as whether she lives close to a bike infrastructure, which may 
+bias our observational study if we're interested in the effect adding a 
+bikelane has on the mode share of bicycles. Those methods, however, rely on 
+collecting attitudinal indicators typically obtained by conducting
+additional surveys with a sample of the people. This may not be an option in 
+many cases; for example, we may not be able to reach the individuals to conduct additional surveys with them. 
+
+
+\subsection{The deconfounder algorithm}
+\label{sec:deconfounder-algo}
+
+
+
+One recent method that has been proposed to deal with the problem of latent 
+confounding without the need to collect additional data is the deconfounder 
+algorithm by \citet{wang_2019_blessings}. The method attempts to control for the 
+confouding variable by estimating a "subsitute confounder", a set of variables 
+that once controlled for, renders all variation in the treatment variables of 
+interest exogenous. The process of applying the method is actually quite 
+straightforward and simple. It proceeds as follows:
+\begin{itemize}
+	\item First, estimate the substitute confounder using any good latent variable 
+	model the modeler chooses. The authors suggest estimating a factor model 
+	with k factors on the set of covariates the modeler is interested in. 
+	\item Second, check the factor model's accuracy using posterior predictive 
+	checks (add details). 
+	\item Once a sufficiently accurate latent variable model is recovered, use it 
+	to estimate an expected value of the latent variable for each observation, 
+	and control for this value in the outcome model, alongside the treatment 
+	variables and other covariates of interest. 
+\end{itemize}
+
+
+
+One main assumption of the deconfounder algorithm is that the data at hand 
+should have multi-cause confounders, thus the title of the paper, "The 
+blessings of multiple causes". In other words, this method works when the 
```

Comment:
Your sentence is totally correct, but I think it's misleading because it doesn't highlight the constraining role of single-cause confounders.
```suggestion
blessings of multiple causes". In other words, this method works when all 
```